packages/expo-ads-admob/ios/EXAdsAdMob.podspec

```
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.source         = { git: 'https://github.com/expo/expo.git' }
 
   s.dependency 'UMCore'
-  s.dependency 'Google-Mobile-Ads-SDK', "7.55.1"
+  s.dependency 'Google-Mobile-Ads-SDK', "8.1.0"
```

```suggestion
  s.dependency 'Google-Mobile-Ads-SDK', "7.69.0"
```

This version is latest miner version of 7.X . 

Ref. https://developers.google.com/admob/ios/rel-notes